framework/auth/oauth_scopes.py

```diff
@@ -62,9 +72,9 @@ class ComposedScopes(object):
                     (CoreScopes.NODE_BASE_WRITE, CoreScopes.NODE_CHILDREN_WRITE, CoreScopes.NODE_LINKS_WRITE)
 
     # Privileges relating to editing content uploaded under that node # TODO: Add wiki etc when implemented
-    NODE_DATA_READ = (CoreScopes.NODE_FILE_READ,)
+    NODE_DATA_READ = (CoreScopes.NODE_FILE_READ, CoreScopes.NODE_COMMENTS_READ)
```

Comment:
No, because not everyone who can see the project can comment on it (e.g. public projects). I think if you have write-permissions to the node you should be able to comment, and if you have read-permissions you can only read comments.